app/main.py

```diff
@@ -2,0 +1,20 @@
+from app.cafe import Cafe
+from app.errors import (
+    VaccineError,
+    NotWearingMaskError
+)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
```

Comment:
Code Style: It's a good practice to use type annotations for all function parameters and return values. For instance, `cafe: Cafe` should be annotated as well.